verifier-agent/node/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18-alpine3.16 as builder
+ENV NODE_ENV development
+WORKDIR /app
+COPY ./ ./
+RUN npm ci --ignore-scripts
+RUN npm run build
```

Comment:
`RUN npm run build && rm -rf node_modules`

Reply:
What happens otherwise? From my understanding `npm ci --only=production` prunes dev dependencies and if it doesn't, could we not use `npm prune --omit=dev`?